README.md

```diff
@@ -94,58 +94,245 @@ git fetch upstream-code main
-	"nameLong": "Red Hat OpenShift Dev Spaces with Microsoft Visual Studio Code - Open Source IDE",
-	"icons": {
-		"favicon": {
-			"universal": "icons/favicon.ico"
-		},
-		"welcome": {
-			"universal": "icons/dev-spaces.svg"
-		},
-		"statusBarItem": {
-			"universal": "icons/dev-spaces.svg"
-		},
-		"letterpress": {
-			"light": "icons/letterpress-light.svg",
-			"dark": "icons/letterpress-light.svg"
-		}
+    "nameShort": "Branded IDE",
+    "nameLong": "Branded Instance of Eclipse Che with Branded Microsoft Visual Studio Code - Open Source IDE",
+    "icons": {
+        "favicon": {
+            "universal": "icons/favicon.ico"
+        },
+        "welcome": {
+            "universal": "icons/icon.svg"
+        },
+        "statusBarItem": {
+            "universal": "icons/icon.svg"
+        },
+        "letterpress": {
+            "light": "icons/letterpress-light.svg",
+            "dark": "icons/letterpress-light.svg"
+        }
+    },
+    "remoteIndicatorCommands": {
+        "openDocumentationCommand": "Branded IDE: Open Documentation",
+        "openDashboardCommand": "Branded IDE: Open Dashboard",
+        "stopWorkspaceCommand": "Branded IDE: Stop Workspace"
+    },
+    "workbenchConfigFilePath": "workbench-config.json",
+    "codiconCssFilePath": "css/codicon.css"
+}
+```
+
+`nameShort` is the application name for UI elements.
+
+`nameLong` is the application name that is used for the **Welcome** page, **About** dialog, and browser tab title.
+
+`favicon` is the icon for the browser tab title for all themes.
+
+`welcome` is the icon for the tab title of the **Get Started** page for all themes.
+
+`statusBarItem` is the icon for the bottom **Status Bar** for all themes. Define it as `codicon` in the `workbench-config.json` file and the `codicon` CSS styles.
+
+`letterpress` is the icon for the empty editor area when no editor is open. You can provide different icon files for `light` and `dark` themes.
+
+`remoteIndicatorCommands` is the names of commands provided by the [Eclipse Che Remote](https://github.com/che-incubator/che-code/blob/main/code/extensions/che-remote/package.nls.json) extension. Users can run these commands by clicking the **Status Bar**.
+
+`workbenchConfigFilePath` is the relative path to `workbench-config.json`, which is explained in one of the next steps.
+
+`codiconCssFilePath` is the relative path to `css/codicon.css`, which is explained in one of the next steps.
+
+:grey_exclamation: The values defined in the `/branding/product.json` file override the [default values](https://github.com/che-incubator/che-code/blob/main/code/product.json).
+
+3\. Add the icon files, which you specified in the `product.json` file in the previous step, to the repository.
+
+4\. Create a `/branding/workbench-config.json` file with custom values.
+
+*Example. `/branding/workbench-config.json`*
+
+```
+{
+	"windowIndicator": {
+		"label": "$(eclipse-che) Eclipse Che",
+		"tooltip": "Eclipse Che"
```

Comment:
It's not clear to me what these two lines are exactly for: the Che instance or the VS Code?

```suggestion
		"label": "$(eclipse-che) Branded IDE",
		"tooltip": "Branded IDE"
```

Reply:
it's responsible for this item:

<img width="238" alt="Screenshot 2022-11-25 at 21 46 07" src="https://user-images.githubusercontent.com/5676062/204049363-98ac1be4-0661-45c7-9f42-4026b1f9c86c.png">

the first part `$(eclipse-che)` - is rendered as the corresponding icon
the next one `Eclipse Che` - as the label
the last one - is tooltip

Reply:
[here](https://github.com/eclipse-che/che-docs/pull/2467/files#diff-f0a52a1e5d88daf1e06a0693ef3331285f726a430a2f802c17b8f1b92324483c) you can see an example for the `Dev Spaces` status bar item when branding was applied

Reply:
Thank you, Roman.
Regarding "label": "$(eclipse-che): 
`(eclipse-che)`--is it `statusBarItem` from line 169? Is this a variable or how does it become a file name for the icon (or which icon file is it)?

Reply:
> is it statusBarItem from line 169?

it's true

>  Is this a variable or how does it become a file name for the icon (or which icon file is it in product.json)?

<img width="1002" alt="Screenshot 2022-11-29 at 07 20 08" src="https://user-images.githubusercontent.com/5676062/204445432-8eebc78c-984f-4bb3-8c68-bd0c7054266d.png">

css file contains style with ending `eclipse-che`.
the file name is defined in the corresponding style.

<img width="425" alt="Screenshot 2022-11-29 at 07 24 47" src="https://user-images.githubusercontent.com/5676062/204446129-c09eb440-103d-429d-9879-5e831ddffdc4.png">

For example, for the `devspaces-code`:
- there is `$devspaces` codicon in [the workbench-config.json file](https://github.com/redhat-developer/devspaces-images/blob/ec9c9776a0f82093616ff0d96385273c8c1cbfbf/devspaces-code/branding/workbench-config.json#L3).
- [css file](https://github.com/redhat-developer/devspaces-images/blob/ec9c9776a0f82093616ff0d96385273c8c1cbfbf/devspaces-code/branding/css/codicon.css#L2) contains the corresponding style with the `devspaces` ending
- the file name (`dev-spaces.svg`) of the icon [is defined](https://github.com/redhat-developer/devspaces-images/blob/ec9c9776a0f82093616ff0d96385273c8c1cbfbf/devspaces-code/branding/css/codicon.css#L3) in the style